zebra/zebra_dplane.h

```diff
@@ -475,6 +485,42 @@ const struct prefix *
 dplane_ctx_rule_get_dst_ip(const struct zebra_dplane_ctx *ctx);
 const struct prefix *
 dplane_ctx_rule_get_old_dst_ip(const struct zebra_dplane_ctx *ctx);
+/* Accessors for policy based routing iptable information */
```

Comment:
Same comment here about attribute-by-attribute accessors: wouldn't it be cleaner to just have a get_pbr_ipset() accessor, a _get_pbr_iptable() accessor, that gets a group of values into the pbr struct at once?

Reply:
again, at the very least, wouldn't it make more sense to have more compact accessors - at least that would encapsulate the code required to init the pbr structs?

Reply:
hmm, no, I didn't mean "get a pointer from inside the context" - I just meant "load up a pbr struct from info inside the context" - like:

```
bool dplane_ctx_get_pbr_iptable(const struct zebra_dplane_ctx *ctx,
                                struct zebra_pbr_iptable *table);
```
You can store the bits and pieces inside the context in various ways, but this lets you extract them in the way that is most meaningful and compact?

Reply:
ok, I tried to reflect this in the new proposed pull request .